application/Profile/LocalProfile.py

```
@@ -37,45 +38,60 @@
+                    value_error = "LocalProfile: Version info not found or mismatch in the profile."
+                    logger.error(value_error)
+                    raise ValueError(value_error)
 
                 self._profile["sessions"] = json_data["sessions"]
 
         except Exception as e:
             self._profile = copy.deepcopy(_EMPTY_USER_PROFILE)
-            print("LocalProfile: load_profile:", e)
-            # raise e
-            print("Unable to load the user profile. Using the default profile instead.")
+            logger.error(f"LocalProfile: Error loading profile: {e}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)